postcss.config.js

```diff
@@ -1,14 +1,20 @@
 const purgecss = require('@fullhuman/postcss-purgecss');
+const cssnano = require('cssnano');
 
-module.exports = {
-  plugins: [
-    require('autoprefixer'),
-    require('cssnano')({
-        preset: 'default',
-    }),
-    purgecss({
-      content: ['./**/*.html'],
-      keyframes: true
-    })
-  ]
+module.exports = (ctx) => {
+  return {
+    plugins: [
+      require('autoprefixer'),
+      ...ctx.options.env === 'production' ? [
+        cssnano({
+          preset: 'default',
+        }),
+        purgecss({
+          content: ['./**/*.html'],
+          keyframes: true,
+          defaultExtractor: content => content.match(/[A-z0-9-:/]+/g)
```

Comment:
What exactly is this line doing?

Reply:
While extracting css purge css extractor skips classes like hover:bg-blue there are classes like this is tailwind css. so i have added regex to overcome that issue. For more info https://github.com/FullHuman/purgecss/pull/183